module/trace/metrics.go

```diff
@@ -1,14 +0,0 @@
-package trace
-
-import (
-	"github.com/prometheus/client_golang/prometheus"
-	"github.com/prometheus/client_golang/prometheus/promauto"
-)
-
-var (
-	spanDurationMetric = promauto.NewHistogramVec(prometheus.HistogramOpts{
```

Comment:
this metric was not exposed before, and I was not sure what would be the purpose as spans might have different sizes